[users] Add a choice to display autologin checkbox

Neixen911 opened the pull request:
By default, the autologin checkbox is display to the user and he can choose if he wants to enable it or not. By adding a variable to display the checkbox or not, this checkbox can be not displaying to the user and avoid him to change it.

Neixen911 committed:
[users] Add a choice to display autologin checkbox

By default, the autologin checkbox is display to the user and he can
choose if he wants to enable it or not. By adding a variable to display
the checkbox or not, this checkbox can be not displaying to the user and
avoid him to change it.

dalto8 commented:
If we want to make this optional, I think we should add a setting in the users.conf file for it.

Neixen911 committed:
[users] Add a setting to displayAutologin

To add this new setting optionnal, a variable has need to be created
in the file users.conf. This variable can be modified and it changes
directly if the autologin checkbox is show or not.

Neixen911 commented:
I added a setting in the users.conf file like you said to make it optional.

dalto8 commented:
Thanks!

The default should be set to match the existing behavior and then distros can override in the config.  Since, this is currently enabled by default, the default should be `true`.

Also, don't forget to update `users.schema.yaml` with the new key.

Neixen911 committed:
[users] Set displayAutologin to true by default

To match the existing behavior, the setting displayAutoLogin have to
be set to true bt default. This new setting is added to
users.schema.yaml too.

Neixen911 commented:
Thanks for your feedback !

I have set the value of displayAutoLogin setting to `true` by default now.

Plus, I updated the file `users.schema.yaml` with this setting too.
